src/StatsdClient/StatsdClient.csproj

```
@@ -4,13 +4,16 @@
     <PackageId>DogStatsD-CSharp-Client</PackageId>
     <Description>A DogStatsD client for C#. DogStatsD is an extension of the StatsD metric server for use with Datadog. For more information visit http://datadoghq.com.</Description>
     <Authors>Datadog</Authors>
-    <TargetFrameworks>netstandard1.3;netstandard1.6;netcoreapp1.1;netcoreapp2.0;net451;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;net451</TargetFrameworks>
```

From [.NET Guide > Open-source library guidance > Cross-platform targeting](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting#multi-targeting):

> CONSIDER adding a target for `net461` when you're offering a `netstandard2.0` target.
>
>Using .NET Standard 2.0 from .NET Framework has some issues that were addressed in .NET Framework 4.7.2. You can improve the experience for developers that are still on .NET Framework 4.6.1 - 4.7.1 by offering them a binary that is built for .NET Framework 4.6.1.

```suggestion
    <TargetFrameworks>netstandard1.3;netstandard2.0;net451;net461</TargetFrameworks>
```